scripts/deploy.sh

```diff
@@ -71,15 +81,21 @@ elif [[ ! -z "$dtag" && -z "$ptag" && -z "$rtag" ]]; then
         exit 1
     fi
 
-    # tag it locally
+    # Merge branch into master and push to origin
+    git checkout master
+    git pull origin
+    git merge --no-ff release_$rtag -m "Merge branch devel_$rtag"
+    git push origin master
+    git branch -d release_$rtag
+
+    # Tag the version locally.
     git tag -a $dtag -m "New devel[rc] build $dtag."
```

Comment:
maybe `"New devel/rc build $dtag."` not sure what square brackets there would signify.